### ISSUE TITLE
Why inf1.xlarge speed is slower than g4.xlarge instance.

### ISSUE THREAD

**Issue by vuquocan1987** [aws/aws-neuron-sdk#208]
I test the inference speed of the the instance. 

I used same environment for both instance "aws_neuron_pytorch_p36" and follow the tutorial: 

"https://github.com/aws/aws-neuron-sdk/blob/master/docs/tensorflow-neuron/tutorial-compile-infer.md"

I was surprise that the speed of the inferentia instance is slower than the G4 instance:

Here is the output:

```
2020-11-23 11:40:50.752194: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
WARNING:tensorflow:From /home/ubuntu/anaconda3/envs/test_conda_env/lib/python3.6/site-packages/tensorflow_core/contrib/predictor/saved_model_predictor.py:153: load (from tensorflow.python.saved_model.loader_impl) is deprecated and will be removed in a future version.
Instructions for updating:
This function will only be available through the v1 compatibility library as tf.compat.v1.saved_model.loader.load or tf.compat.v1.saved_model.load. There will be a new function for importing SavedModels in Tensorflow 2.0.
[('n02123045', 'tabby', 0.6956522), ('n02127052', 'lynx', 0.120923914), ('n02123159', 'tiger_cat', 0.08831522), ('n02124075', 'Egyptian_cat', 0.06453805), ('n02128757', 'snow_leopard', 0.009341033)]
3.3114852905273438
```

3.3 seconds is the time that I measure on inf1.xlarge instance, while on the G4 instance it was only about 1 seconds. 

It was advertised on the aws site that inf1 is supposed to be faster? Am I missing something in the set up?

Here is the output for the compiling code:

```
2020-11-23 11:35:01.511943: I tensorflow/core/grappler/optimizers/meta_optimizer.cc:788]   aws_neuron_fuse_supported_operators: Graph size after: 3 nodes (-872), 2 edges (-888), time = 76.619ms.
INFO:tensorflow:fusing subgraph {subgraph neuron_op_d6f098c01c780733 with input tensors [<tf.Tensor 'input_10/_0:0' shape=(1, 224, 224, 3) dtype=float32>], output tensors [<tf.Tensor 'probs/Softmax:0' shape=(1, 1000) dtype=float32>]} with neuron-cc; you may check progress by inspecting file /tmp/tmpahinaukc/neuron_op_d6f098c01c780733/graph_def.neuron-cc.log
INFO:tensorflow:Number of operations in TensorFlow session: 4638
INFO:tensorflow:Number of operations after tf.neuron optimizations: 876
INFO:tensorflow:Number of operations placed on Neuron runtime: 874
INFO:tensorflow:Successfully converted ./ws_resnet50/resnet50 to ./ws_resnet50/resnet50_neuron
```

**Comment by awshaichen** [comment]
Hello,

This tutorial is meant for demonstrating how to write a minimal save/compile example, rather than inference performance. In fact, the timings you are seeing so far are model warm-up times on Inferentia as well as on GPU, and you could time the second inference instead of the first one to get much better performance on both hardwares.

Also, please refer to https://github.com/aws/aws-neuron-sdk/tree/master/src/examples/tensorflow/keras_resnet50 as a self-contained example for fully optimizing the performance of ResNet50 and deploying it on AWS inf1 instances.

Thanks,
Haichen

**Comment by vuquocan1987** [comment]
Thank you very much. This certainly answer my question.
